client/utils/countryCodes.js

```diff
@@ -0,0 +1,230 @@
+export default {
```

Comment:
Ideally, this will be a real database, but yeah, I'm fine front-loading it before we get the schema updated.